pom.xml

```diff
@@ -874,6 +891,12 @@ THE SOFTWARE.
                     <bannedImport>org.apache.commons.httpclient.**</bannedImport>
                   </bannedImports>
                 </restrictImports>
+                <bannedDependencies>
+                  <excludes>
+                    <exclude>org.apache.commons:commons-lang3</exclude>
+                  </excludes>
+                  <searchTransitive>false</searchTransitive>
```

Comment:
use `true` here or remove this as `true` is default.

Reply:
`true` will cause build failure because `commons-lang3-api` depends on `org.apache.commons:commons-lang3`

```suggestion
```